scripts/cc-cli/src/commands/unbond.ts

```diff
@@ -23,7 +18,7 @@ async function unbondAction(options: OptionValues) {
   checkAmount(options);
 
   // Build account
-  const seed = getSeedFromOptions(options);
+  const seed = await getStashSeedFromEnvOrPrompt();
```

Comment:
@pLabarta please correct me if I'm wrong but isn't the validator supposed to use the Stash account only when bonding and use the Controller account for everything else?  Here it is the opposite.

Can you audit this PR to make sure it is using the correct accounts?

Reply:
Just adding these here for reference:
https://github.com/paritytech/substrate/blob/master/frame/staking/src/pallet/impls.rs#L1571

Reply:
Will update the code to match

Reply:
done